### PR TITLE
Add labs flag for alternate message layouts

### PR DESCRIPTION
**Pull request by JorikSchellekens**
Impelemts setting for https://github.com/matrix-org/matrix-react-sdk/pull/4531

### CODE REVIEW

File: docs/labs.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ dropped. Ask in the room if you are unclear about any details here.**
 Enables font scaling options for accessibility. To alter the scale check the
 appearance tab in settings.
 
+## Alternate message layouts (`feature_alternate_message_layouts`)
+
+Enables options in rooms settings to switch between message layouts such as the default and "irc-like" layouts.
```
Comment (turt2live) [inline docs/labs.md:17]: room settings sounds like the wrong place to put this

Comment (JorikSchellekens) [reply docs/labs.md]: Good point, it's now a labs setting which covers all rooms

**Comment by JorikSchellekens** [comment]
The partner pr is up fyi

**Comment by turt2live** [comment]
ftr: generally a multi-part PR shouldn't land in pieces this far apart without approving reviews on all sides. Also not sure what happened to have caused this one to be blocked on my review but the actual functionality to end up in the main queue, but that might be for a retro.

**Comment by turt2live** [comment]
> need a riot-desktop PR now

https://github.com/vector-im/riot-desktop/pull/88

**Pull request merged**